WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/HelpAndSupportViewController.swift

```diff
@@ -168,6 +225,18 @@ private extension HelpAndSupportViewController {
         )
     }
 
+    /// Contact WCPay Support cell.
+    ///
+    func configureContactWCPaySupport(cell: ValueOneTableViewCell) {
+        cell.accessoryType = .disclosureIndicator
+        cell.selectionStyle = .default
+        cell.textLabel?.text = NSLocalizedString("Contact WCPay Support", comment: "Contact WCPay Support title")
+        cell.detailTextLabel?.text = NSLocalizedString(
+            "Reach our happiness engineers who can help answer tough questions",
```

Comment:
I'm working on the same ticket. Shall we emphasize the payments here?

```
Reach our happiness engineers who can help answer payments related questions
```
maybe?

Reply:
Yes - I like that. cc @Garance91540  and @adamzelinski

Reply:
@kidinov @allendav yes please, let's use that copy :)